src/dimension.c

```diff
@@ -1808,7 +1809,9 @@ Datum
 ts_hash_dimension(PG_FUNCTION_ARGS)
 {
 	Ensure(PG_NARGS() > 2, "expected at most 3 arguments, invoked with %d arguments", PG_NARGS());
-	DimensionInfo *info = make_dimension_info(PG_GETARG_NAME(0), DIMENSION_TYPE_CLOSED);
+	Name column_name;
+	GETARG_NOTNULL_NULLABLE(column_name, 0, "column name", NAME);
```

Comment:
```suggestion
	GETARG_NOTNULL_NULLABLE(column_name, 0, "column_name", NAME);
```
This should match the parameter name